lib/active_remote/persistence.rb

```diff
@@ -241,6 +241,7 @@ def remote_create
 
         response = rpc.execute(:create, new_attributes)
 
+        reset_attributes
```

Comment:
One thing to consider: I was debating whether or not we should have some conditional logic here to maybe not reset if the response contains errors and the response attributes are empty; however, we should never expect to receive an empty attribute set. The only situation where we shouldn't expect to have any attributes is if there's some kind of error raised back through active remote, but that should happen before we get to this point.